### ISSUE TITLE
Suggestion regarding the hotkey Pass next key through

### ISSUE THREAD

**Issue by LittleStar-VIP** [nvaccess/nvda#13749]
### Is your feature request related to a problem? Please describe.
Some of the keyboard hotkeys on Google drive, Gmail etc involve more than one key, e.g. first pressing g then another key.
As the nvda function only allow the next key to be passed through by pressing nvda+f2, we cannot utilize those hotkeys from Google Drive, Gmail, etc.

### Describe the solution you'd like
Probably user can define a time limit, say 3 seconds.
Within the time limit, those key press will all be passed through

**Comment by lukaszgo1** [comment]
In browse mode you can temporarily disable quick navigation keys with NVDA+Shift+Space. Is this insufficient for you?

**Comment by LittleStar-VIP** [comment]
@lukaszgo1 
Oh, I do not know there is such a hotkey, thank you for letting me know, let me try using this one.

**Comment by cary-rowen** [comment]
If NVDA + Shift + Space works for you please close this issue
